downloads/index.md

```diff
@@ -1,30 +1,20 @@
 # Download Julia
 
-If you like Julia, please consider starring us [on GitHub](https://github.com/JuliaLang/julia) and spreading the word!
-
 ~~~
 <a class="github-button" href="https://github.com/JuliaLang/julia" data-size="large" data-show-count="true" aria-label="Star JuliaLang/julia on GitHub">Star</a>
 ~~~
 
+Please star us [on GitHub](https://github.com/JuliaLang/julia). If you use Julia for your research, please do [cite us](https://julialang.org/research/).
+
 We provide several ways for you to run Julia:
 
 * In the terminal using the built-in Julia command line using the binaries provided below.
```

Comment:
```suggestion
* In the terminal using the built-in Julia command (download binaries provided below).
* In VS Code via the [Julia VS Code Extension](https://code.visualstudio.com/docs/languages/julia#_getting-started) (download binaries provided below)
```

Reply:
This isn't totally complete since we only mention VS Code but seems right to add it to the list there

Reply:
The VS code extension does not provide a Julia download. This is not the right place for mentioning it.